example/sleep.har

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
I get `Invalid page sleep (0): must be array` when trying to generate k6 script from this file.

Reply:
Yes this one should be replaced. This is the draft 1 version